[2022-06-30 00:02:27,754] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_C scheduled__2022-06-29T00:00:00+00:00 [queued]>
[2022-06-30 00:02:27,768] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_C scheduled__2022-06-29T00:00:00+00:00 [queued]>
[2022-06-30 00:02:27,770] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-30 00:02:27,771] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-30 00:02:27,772] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-30 00:02:27,793] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_C> on 2022-06-29 00:00:00+00:00
[2022-06-30 00:02:27,797] {standard_task_runner.py:52} INFO - Started process 16098 to run task
[2022-06-30 00:02:27,800] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_C', 'scheduled__2022-06-29T00:00:00+00:00', '--job-id', '914', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpjvcuiyk_', '--error-file', '/tmp/tmp4o8vo8ih']
[2022-06-30 00:02:27,801] {standard_task_runner.py:80} INFO - Job 914: Subtask model_C
[2022-06-30 00:02:27,877] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_C scheduled__2022-06-29T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-30 00:02:28,037] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_C
AIRFLOW_CTX_EXECUTION_DATE=2022-06-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-29T00:00:00+00:00
[2022-06-30 00:02:28,041] {python.py:173} INFO - Done. Returned value was: 6
[2022-06-30 00:02:28,100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_C, execution_date=20220629T000000, start_date=20220630T000227, end_date=20220630T000228
[2022-06-30 00:02:28,145] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-30 00:02:28,213] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
